pkg/trace/info/stats.go

```diff
@@ -204,6 +204,11 @@ type TracesDropped struct {
 	SpanIDZero int64
 	// ForeignSpan is when a span in a trace has a TraceId that is different than the first span in the trace
 	ForeignSpan int64
+	// Timeout is when a request times out
```

Comment:
```suggestion
	// Timeout counts the number of payloads dropped due to read timeouts.
```

Reply:
Sure, but it's using the same sentence structure as all the other ones?

Reply:
Yeah, I share the sentiment. Unfortunately that's the thing with this codebase. Some of it is older, some of it is newer. I'm trying to aim and make sure that at the very least the new code respects common idioms. Please consider this when seeing my comments, even if you copied code from elsewhere that might not. If you'd like, it's fine to change it everywhere but I wouldn't want to ask that of you.